templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -297,7 +297,9 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.    # intermediate saving of cell data with SOM and meta labels assigned
</pre><p>I think this is a leftover comment?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/999/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>